web/src/people.ts

```diff
@@ -41,6 +41,8 @@ export type User = {
     profile_data: Record<number, ProfileData>;
     is_missing_server_data?: boolean; // used for fake user objects.
     is_inaccessible_user?: boolean; // used for inaccessible user objects.
+    // Used in `typeahead_helper`
+    special_item_text?: string;
```

Comment:
I thought you were arranging to avoid any changes to this file? This is not the correct type for a `User`.

Reply:
Oh I see, this is part of the pseudo mention user. I'll fix that, thanks!